src/components/VideoDetails/RelatedCard.jsx

```diff
@@ -1,30 +1,30 @@
-import React from 'react'
+import React from 'react';
 import styled from 'styled-components';
 import { useHistory } from 'react-router-dom';
 import { Grid } from '@material-ui/core';
 
 const StyledRelatedCard = styled.div`
-cursor: pointer;
+  cursor: pointer;
 `;
 
 const StyledImd = styled.img`
-width: 100%;
+  width: 100%;
 `;
```

Comment:
Extract these styled components to the respective `styled.js` file, so that you have a cleaner `RelatedCard` component.